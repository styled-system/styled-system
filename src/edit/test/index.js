import React from 'react'
import renderer from 'react-test-renderer'
import {
  EditProvider,
  Field,
  FieldSet,
  ThemeControls,
  ResetButton,
} from '../src'

const theme = {
  colors: {
    text: 'black',
    background: 'white',
  },
  fontSizes: [ 12, 14, 16, 20, 24, 32 ],
}

test('renders', () => {
  const json = renderer.create(
    <EditProvider initialTheme={theme}>
      <ThemeControls />
    </EditProvider>
  ).toJSON()
  expect(json).toMatchSnapshot()
})
