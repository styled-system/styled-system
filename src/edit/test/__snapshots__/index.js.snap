// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="css-1dt9r02-ThemeControls"
/>
`;
